llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -552,6 +515,16 @@ createScalarIVSteps(VPlan &Plan, InductionDescriptor::InductionKind Kind,
   return Steps;
 }
 
+/// Return the header mask recipe of the VPlan, if there is one.
+static VPInstruction *getHeaderMask(VPlan &Plan) {
```

Comment:
Having loop region provide getHeaderMask() should save this search for a singleton VPValue.